addons/l10n_cz/models/account_move.py

```diff
@@ -5,4 +5,15 @@
 class AccountMove(models.Model):
     _inherit = 'account.move'
 
-    taxable_supply_date = fields.Date()
+    taxable_supply_date = fields.Date(string="Taxable Supply Date", default=fields.Date.context_today, required=True)
```

Comment:
Targetting stable and adding some store fields :grimacing: i think we will need a bridge for all those fields :smile: